concept-api/src/main/scala/no/ndla/conceptapi/model/domain/Concept.scala

```diff
@@ -39,7 +39,8 @@ case class Concept(
     visualElement: Seq[VisualElement],
     responsible: Option[Responsible],
     conceptType: ConceptType.Value,
-    glossData: Option[GlossData]
+    glossData: Option[GlossData],
+    editorNote: Seq[EditorNote]
```

Comment:
Er tross alt ei liste
```suggestion
    editorNotes: Seq[EditorNote]
```